tests/test_algorithm/test_get_conditional_independencies.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+"""Test getting conditional independencies."""
+
+import unittest
+from typing import Set
+
+from y0.algorithm.conditional_independencies import ConditionalIndependency, get_conditional_independencies
+from y0.graph import NxMixedGraph, napkin_graph
+
+
+class TestGetConditionalIndependencies(unittest.TestCase):
```

Comment:
other tests welcome! Likely better to follow #26 when more graphs are available and this PR won't blow up

Reply:
Done. now tests can be arbitrarily added as instances of the `Example` class in `y0.examples`